### DISCUSSION TITLE
Problems

### DISCUSSION THREAD

**Discussion by johnrobledo** [fuatakgun/eufy_security#96] [Q&A] [answered]
Stream takes forever to buffer, even with RTSP. Sometimes it won't load at all, just keep buffering. HELP! Would like to know when the thumbnail updates and if I can manually set the time interval for it to update. If it updates by motion, that is fine. PS. the eufy security addon is showing some error logs

**Comment by fuatakgun** [comment, marked as answer]
https://github.com/fuatakgun/eufy_security/issues/102
I have the same issue and using webrtc to avoid this. I do not have a solution in place yet.
